README.es.md

```diff
@@ -2394,13 +2413,13 @@ set :protection, :except => [:path_traversal, :session_hijacking]
   <dt>server</dt>
   <dd>
     Servidor, o lista de servidores, para usar como servidor
-    integrado. Por defecto: <tt>['thin', 'mongrel', 'webrick']</tt>,
```

Comment:
This line was behind the current documentation where thin, mongrel nor webrick are not mentioned.